### PR TITLE
Standardised reference data section

### PR DESCRIPTION
**Pull request by chfoster**
Added new section to explain the process of using links to GitHub raw files. 

HOWEVER, I've noticed an issue #8  that using the standard [...](#section-name) code to link to sections that are not within that .qmd script doesn't work! 

A previous example that works on the old markdown page was [SQL learning page](sql.html#Givinggetting_access) however this now doesn't work in quarto - it's trying to find sql.html within the creating statistics folder still which isn't where it sits - it sits in a learning development folder.

### CODE REVIEW

File: RAP/rap.qmd [hunk 1]
```diff
@@ -245,6 +245,36 @@ Much like setting up a SQL area, there are different ways to do this depending o
...
+
+There will of course be cases where some of the data you use is reference data not owned by the DfE, or is available online for you to download rather than in an internal server. There are ways of incorporating this into a reproducible analytical pipeline nicely, sometimes you can even use links/URLs in your code that will always pull the latest data such that you will never need to change the link to reflect updates to the data!
+
+---
+
+### Raw files on GitHub
+
+---
+
+If you want to use reference data from GitHub, you can use the URL to the *raw* file. A common example of when you might want to use this would be to use **geographic lookup tables** that contain names and codes of different geographic levels for EES files (which are available in our [data screener repository](https://github.com/dfe-analytical-services/dfe-published-data-qa)).
```
Comment (cjrace) [inline RAP/rap.qmd]: Could we make this open in a new tab?

File: creating-statistics/ud.qmd [hunk 2]
```diff
@@ -608,6 +608,8 @@ The geographic_level column should describe the level of data present in that ro
 
 Teams should make sure that they are regularly checking their geography codes if they are not using a lookup from a maintained database (such as in the PDR). ONS have the [Open Geography portal](http://geoportal.statistics.gov.uk/){target="_blank" rel="noopener noreferrer"}, which can be a useful way of checking these. There is a wealth of data on there, though Local authority boundaries can be hard to find, they can be found using the tabs at the top – Boundaries > Administrative Boundaries > Counties and Unitary Authorities.
 
+Teams can also use the lookup tables available via the [data screener GitHub repository data folder](https://github.com/dfe-analytical-services/dfe-published-data-qa/tree/master/data). Guidance on how to link directly to the latest data in your code can be seen in the [standardised reference data section](rap.html#Standardised_reference_data)
```
Comment (cjrace) [inline creating-statistics/ud.qmd]: Make data screener link to github open in new tab, internal link to RAP is fine as is

**Review by cjrace — CHANGES_REQUESTED** [review]

**Comment by chfoster** [comment]
close #8 with latest commit fix :)

File: RAP/rap.qmd [hunk 3]
```diff
@@ -1162,4 +1192,4 @@ git clone YOUR_URL_HERE
 
 - Make use of git and version control in your team projects regularly. Like learning anything new, putting it into practice regularly is the best way to become confident in using it.  
 
-Please refer to the other links on the [learning resources](l+d.html#git) page to learn more about how to use git in practice.
+Please refer to the other links on the [learning resources](../learning-development/learning-development.html#git) page to learn more about how to use git in practice.
```
Comment (cjrace) [inline RAP/rap.qmd]: This link doesn't work? Should it just go straight to the git page? (Tbh I suspect the old link didn't work either so it's not a new break...)

File: creating-statistics/ud.qmd [hunk 4]
```diff
@@ -608,6 +608,8 @@ The geographic_level column should describe the level of data present in that ro
 
 Teams should make sure that they are regularly checking their geography codes if they are not using a lookup from a maintained database (such as in the PDR). ONS have the [Open Geography portal](http://geoportal.statistics.gov.uk/){target="_blank" rel="noopener noreferrer"}, which can be a useful way of checking these. There is a wealth of data on there, though Local authority boundaries can be hard to find, they can be found using the tabs at the top – Boundaries > Administrative Boundaries > Counties and Unitary Authorities.
 
+Teams can also use the lookup tables available via the [data screener GitHub repository data folder](https://github.com/dfe-analytical-services/dfe-published-data-qa/tree/master/data){target="_blank" rel="noopener noreferrer"}. Guidance on how to link directly to the latest data in your code can be seen in the [standardised reference data section](../RAP/rap.html#Standardised_reference_data)
```
Comment (cjrace) [inline creating-statistics/ud.qmd:611]: Make the GitHub link a new tab (any external = new tab, anything internal is fine)

File: learning-development/git.qmd [hunk 5]
```diff
@@ -97,7 +97,7 @@ However, you can still run the full suite of git commands by simply typing them
 
 - For those wanting to go deeper into understand the variety of git commands and what they do, there is a great [online visual resource](https://dev.to/lydiahallie/cs-visualized-useful-git-commands-37p1){target="_blank" rel="noopener noreferrer"}.
 
-- We also have a number of helpful sections on using git in practice at the end of our [RAP page](rap.html#Build_capability_within_the_team){target="_blank" rel="noopener noreferrer"}.
+- We also have a number of helpful sections on using git in practice at the end of our [RAP page](../RAP/rap.html#Build_capability_within_the_team){target="_blank" rel="noopener noreferrer"}.
```
Comment (cjrace) [inline learning-development/git.qmd]: This doesn't need to be a new tab

File: learning-development/learning-development.qmd [hunk 6]
```diff
@@ -9,7 +9,7 @@ title: "Learning and Development"
 
 As statistics producers we require a variety of tools to be able to process our data. Below are the recommended tools that will give us the most power to do what we need. These have large user communities in DfE, and are already working in our current IT setup.
 
-For best practice when using software and coding in our process, see our [guidance on RAP](rap.html){target="_blank" rel="noopener noreferrer"} the [DfE Good Code Practice guide](https://dfe-analytical-services.github.io/good-code-practice/index.html){target="_blank" rel="noopener noreferrer"}.
+For best practice when using software and coding in our process, see our [guidance on RAP](../RAP/rap.html){target="_blank" rel="noopener noreferrer"} the [DfE Good Code Practice guide](https://dfe-analytical-services.github.io/good-code-practice/index.html){target="_blank" rel="noopener noreferrer"}.
```
Comment (cjrace) [inline learning-development/learning-development.qmd]: These don't need to be new tabs

File: learning-development/learning-development.qmd [hunk 7]
```diff
@@ -69,7 +69,7 @@ The time commitment of the programme will be dependent on the size of the work a
 
 ---
 
-For more information on how to use these tools in your production processes, please see the [processes and RAP](rap.html){target="_blank" rel="noopener noreferrer"} page.
+For more information on how to use these tools in your production processes, please see the [processes and RAP](../RAP/rap.html){target="_blank" rel="noopener noreferrer"} page.
```
Comment (cjrace) [inline learning-development/learning-development.qmd]: This doesn't need to be a new tab

File: publishing-statistics/dashboards.qmd [hunk 8]
```diff
@@ -469,7 +469,7 @@ The [SCAP LA school places scorecards app](https://department-for-education.shin
 
 To ensure that they are reliable, dashboards should have an appropriate amount of automated tests before they can be published. We recommend using a mix of UI and unit tests, but the number and type of tests that you need to run will depend on the content of your application.
 
-Further guidance on setting up testing can be found in the [DfE good code practice guide](https://dfe-analytical-services.github.io/good-code-practice/testing.html#testing-shiny-apps). Also see our guidance on testing R code on the [learning resources page](l+d.html#R).
+Further guidance on setting up testing can be found in the [DfE good code practice guide](https://dfe-analytical-services.github.io/good-code-practice/testing.html#testing-shiny-apps). Also see our guidance on testing R code on the [learning resources page](../learning-development/l+d.html#R).
```
Comment (cjrace) [inline publishing-statistics/dashboards.qmd]: First link should be a new tab

**Review by cjrace — CHANGES_REQUESTED** [review]

File: creating-statistics/ud.qmd (hunk 4 above)
Comment (chfoster) [reply creating-statistics/ud.qmd:611]: It is isn't it?

Comment (cjrace) [reply creating-statistics/ud.qmd:611]: My bad, it looks like it is!

**Review by cjrace — APPROVED** [review]

**Pull request merged**